endpoints/getting-started/src/main/appengine/swagger.yaml

```diff
@@ -0,0 +1,106 @@
+swagger: "2.0"
+info:
+  description: "A simple Google Cloud Endpoints API example."
+  title: "Endpoints Example"
+  version: "1.0.0"
+host: "YOUR-PROJECT-ID.appspot.com"
```

Comment:
The quickstart linked in the README points this out. Our goal was to keep the README as short and simple as possible, linking the user to all the information he needs.